github.js

```diff
@@ -117,32 +115,15 @@ var GithubLocation = function(options, ui) {
 
   this.ui = ui;
 
-  this.execOpt = {
-    cwd: options.tmpDir,
-    timeout: options.timeout * 1000,
-    killSignal: 'SIGKILL',
-    maxBuffer: this.max_repo_size || 2 * 1024 * 1024,
-    env: extend({}, process.env)
-  };
-
   this.defaultRequestOptions = {
     strictSSL: 'strictSSL' in options ? options.strictSSL : true
   };
 
-  if (!this.defaultRequestOptions.strictSSL) {
-    this.execOpt.env.GIT_SSL_NO_VERIFY = '1'
-  }
-
-  var self = this, envMap = {
-    ca: 'GIT_SSL_CAINFO',
-    cert: 'GIT_SSL_CERT',
-    key: 'GIT_SSL_KEY'
```

Comment:
We're no longer `exec`ing, so this make no sense now.

On Sep 6, 2016 12:26, "Guy Bedford" <notifications@github.com> wrote:

> In github.js
> <https://github.com/jspm/github/pull/107#discussion_r77669492>:
>
> >    this.defaultRequestOptions = {
> >      strictSSL: 'strictSSL' in options ? options.strictSSL : true
> >    };
> >
> > -  if (!this.defaultRequestOptions.strictSSL) {
> > -    this.execOpt.env.GIT_SSL_NO_VERIFY = '1'
> > -  }
> > -
> > -  var self = this, envMap = {
> > -    ca: 'GIT_SSL_CAINFO',
> > -    cert: 'GIT_SSL_CERT',
> > -    key: 'GIT_SSL_KEY'
>
> Can you justify deciding to remove support for this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jspm/github/pull/107/files/35ce9c9ae8aeafb727b9467ef19c09f398323dbc#r77669492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdsPLU1rEx9iaRUFTNoaXjwLsMBCTaoks5qnZQlgaJpZM4J2AfN>
> .
>